# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../strace/temp-compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- strace-4.5.12/resource.c.vanilla	2005-10-14 15:47:08.000000000 +0200
+++ strace-4.5.12/resource.c	2005-10-14 15:51:22.000000000 +0200
@@ -36,6 +36,7 @@
 #ifdef LINUX
 #include <sys/times.h>
 #include <linux/kernel.h>
+#define _LINUX_QUOTA_VERSION 1
 #include <sys/quota.h>
 #endif /* LINUX */
 #ifdef SUNOS4
